bgbb/sql/sql_utils.py

```diff
@@ -3,7 +3,7 @@
 
 import pandas as pd
 
-from bgbb.sql.sql_utils_tests import test_ho_range, test_model_range, test_rft
+# from bgbb.sql.sql_utils_tests import test_rft
```

Comment:
Leave it out if it's not being used.

Reply:
Looks like there's an instance of it in this file -- maybe a good idea to leave uncommented. It also means that the test coverage could be higher to catch issues like that.